vermouth/tests/test_dssp.py

```diff
@@ -361,6 +364,37 @@ def test_run_dssp(savefile, tmpdir):
         assert not os.listdir(str(tmpdir))
 
 
+@pytest.mark.parametrize('pdb, loglevel,expected', [
+    (PDB_PROTEIN, 10, True),  # DEBUG
```

Comment:
It would be better to have the log level constants rather than their underlying numerical values.